resources/user-guides/attach-data.md

```diff
@@ -181,24 +186,24 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### Attaching additional Target request objects to Target to prefetch content events
+### Attaching additional data Target to prefetch content events<a name="Target-Attach-Data-Prefetch-Content"></a>
```

Comment:
Please update the heading as follows:
"Attach additional data to Target to prefetch content events"